pkg/api/validation/validation.go

```diff
@@ -30,12 +30,32 @@ import (
 	"github.com/golang/glog"
 )
 
+const minPort int = 1
+const minHostPort int = 0
+const maxPort int = 65536
+
+const QualifiedNameErrorMsg string = "must match regex [" + util.Dns1123SubdomainFmt + " / ] " + util.Dns1123LabelFmt
```

Comment:
They are used in tests, so I thought they do. But it seems that they are visible inside a package even if not exported. Need to learn more Go...